### ISSUE TITLE
Filenames with dashes produce unquoted table names

### ISSUE THREAD

**Issue by sethfitz** [AppGeo/into-cartodb#10]
...which leads to 400s from the CartoDB API.

e.g.:

``` bash
into-cartodb -k <redacted> -u <redacted> -f whatever-2m.csv
```

`whatever-2m` is treated as the table name, but it's not being quoted somewhere.

```
Unhandled rejection Error: {"error":["syntax error at or near \"-\""]}
    at PassThrough.<anonymous> (/private/tmp/cartodb/node_modules/into-cartodb/node_modules/cartodb-tools/lib/index.js:205:27)
    at emitNone (events.js:72:20)
    at PassThrough.emit (events.js:166:7)
    at finishMaybe (/private/tmp/cartodb/node_modules/into-cartodb/node_modules/readable-stream/lib/_stream_writable.js:509:14)
    at afterWrite (/private/tmp/cartodb/node_modules/into-cartodb/node_modules/readable-stream/lib/_stream_writable.js:388:3)
    at afterTick (/private/tmp/cartodb/node_modules/into-cartodb/node_modules/readable-stream/node_modules/process-nextick-args/index.js:11:8)
    at doNTCallback0 (node.js:417:9)
    at process._tickCallback (node.js:346:13)
```

**Comment by calvinmetcalf** [comment]
I bet it's hear https://github.com/AppGeo/into-cartodb/blob/f1f027ebce6e8400dfb6fce3b847affbbd1c4866/index.js#L89-L94

**Comment by sethfitz** [comment]
Looks likely.

**Comment by calvinmetcalf** [comment]
should be fixed in v1.7.1
